examples/cryptocurrency-advanced/backend/src/lib.rs

```diff
@@ -39,11 +39,12 @@ pub mod schema;
 pub mod transactions;
 pub mod wallet;
 
+use self::failure::Error as EncodingError;
```

Comment:
What the following line do? Why not to use `failure::Error` directly?

Reply:
Yeah, this is WIP implementation that at least compiles, It is still not refactored. 
Thanks for comments, btw.